pom.xml

```diff
@@ -76,6 +76,10 @@
 			<version>${junit.jupiter.version}</version>
 			<scope>test</scope>
 		</dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
```

Comment:
Afaik, we don't really need this dependency. Please inspect it and if indeed not needed, revert this line of code.